solutions/usaco-639.mdx

```diff
@@ -50,4 +50,32 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+```py
+fin = open('diamond.in', 'r')
+
+N, K = map(int, fin.readline().split())
+
+diamonds = []
+
+for _ in range(N):
+    diamonds.append(int(fin.readline()))
+
+tracker = []
+largest, curlargest = 0
+
+for x in range(N):
+    for y in range(N):
+        if diamonds[y] >= diamonds[x] and diamonds[y] <= diamonds[x] + K:
+            curlargest += 1
+    largest = max(largest, curlargest)
+    curlargest = 0
+
+with open('diamond.out', 'w') as fout:
+    fout.write(str(largest) + '\n')
```

Comment:
just saying, you could make this more concise by writing
```py
print(largest, file=open('diamond.out', 'w'))
```

Reply:
ohhhh ok, adding it.

Reply:
done. thanks for the suggestion, i'll start using that now. :)